tests/e2e/test_rosacli_cluster.go

```diff
@@ -1743,3 +1744,143 @@ var _ = Describe("create/delete operator-roles and oidc-provider to cluster",
 				}
 			})
 	})
+var _ = Describe("Reusing opeartor prefix and oidc config to create clsuter", labels.Feature.Cluster, func() {
+	defer GinkgoRecover()
+	var (
+		rosaClient               *rosacli.Client
+		profile                  *profilehandler.Profile
+		err                      error
+		oidcConfigToClean        string
+		ocmResourceService       rosacli.OCMResourceService
+		originalMajorMinorVerson string
+		clusterConfig            *config.ClusterConfig
+		clusterService           rosacli.ClusterService
+		awsClient                *aws_client.AWSClient
+		operatorPolicyArn        string
+		clusterID                string
+	)
+	const versionTagName = "rosa_openshift_version"
+
+	BeforeEach(func() {
+		By("Init the client")
+		rosaClient = rosacli.NewClient()
+		ocmResourceService = rosaClient.OCMResource
+		clusterService = rosaClient.Cluster
+		profile = profilehandler.LoadProfileYamlFileByENV()
+		Expect(err).ToNot(HaveOccurred())
+		clusterConfig, err = config.ParseClusterProfile()
+		Expect(err).ToNot(HaveOccurred())
+
+		awsClient, err = aws_client.CreateAWSClient("", "")
+		Expect(err).To(BeNil())
+
+		By("Get the cluster")
+		clusterID = config.GetClusterID()
+		Expect(clusterID).ToNot(Equal(""), "ClusterID is required. Please export CLUSTER_ID")
+	})
+
+	AfterEach(func() {
+		By("Recover the operator role policy version")
+		keysToUntag := []string{versionTagName}
+		err = awsClient.UntagPolicy(operatorPolicyArn, keysToUntag)
+		Expect(err).To(BeNil())
+		tags := map[string]string{versionTagName: originalMajorMinorVerson}
+		err = awsClient.TagPolicy(operatorPolicyArn, tags)
+		Expect(err).To(BeNil())
+
+		By("Delete resources for testing")
+		output, err := ocmResourceService.DeleteOIDCConfig(
+			"--oidc-config-id", oidcConfigToClean,
+			"--mode", "auto",
+			"-y",
+		)
+		Expect(err).To(BeNil())
+		textData := rosaClient.Parser.TextData.Input(output).Parse().Tip()
+		Expect(textData).To(ContainSubstring("Successfully deleted the OIDC provider"))
+
+	})
+
+	It("to reuse operator-roles prefix and oidc config - [id:60688]",
+		labels.Critical, labels.Runtime.Day2,
```

Comment:
Is it a critical scenario in the customer usage?

Reply:
Yes it it